### PR TITLE
feat: expose rackID option in ingester samara client 

### PR DESCRIPTION
**Pull request by shyimo**
Signed-off-by: shaimoria <shai.moria@zooz.com>

## Which problem is this PR solving?
It is common to have a Kafka cluster spanning multiple datacenters. For example, a common deployment is within an AWS region in which each availability zone is treated as a datacenter. In order to reduce cross region/rack network cost, we can tell the consumer which rack he's located on. this way the consumer will fetch only from partitions that are located on he's region/rack 

## Short description of the changes
- expose `kafka.consumer.rack-id` cli argument in order to ingester consumers to fetch from closest replica

### CODE REVIEW

File: cmd/ingester/app/flags.go [hunk 1]
```diff
@@ -111,6 +113,10 @@ func AddFlags(flagSet *flag.FlagSet) {
 		KafkaConsumerConfigPrefix+SuffixEncoding,
 		DefaultEncoding,
 		fmt.Sprintf(`The encoding of spans ("%s") consumed from kafka`, strings.Join(kafka.AllEncodings, "\", \"")))
+	flagSet.String(
+		KafkaConsumerConfigPrefix+SuffixRackID,
+		"",
+		"Configure the 'rack' in which the consumer resides to enable - must be supported by kafka server")
```
Comment (yurishkuro) [inline cmd/ingester/app/flags.go:119]: Incomplete sentence

**Review by yurishkuro — COMMENTED** [review]
Please explain the use case in the PR description

File: cmd/ingester/app/flags.go (hunk 1 above)
Comment (shyimo) [reply cmd/ingester/app/flags.go]: Hi @yurishkuro.
i update the description

**Comment by shyimo** [comment]
@yurishkuro add explanation for the use case

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**